server/documents/modules/calendar.html.eco

```diff
@@ -339,6 +339,8 @@ themes      : ['Default']
     <div class="example" data-since="2.9.0">
       <h4 class="ui header">Custom format</h4>
       <p>You can entirely change the date format by using a token string formatter.</p>
+      <p>The format is applied to several date outputs mentioned as tokens in the <a href="#/settings">formatter settings</a>, like calendar cells.</p>
```

Comment:
This is unfortunately still wrong, because especially the  `formatter.cell` setting can actually _not_ be used as token string

 ```suggestion
       <p>The format is applied to several date outputs mentioned as tokens in the <a href="#/settings">formatter settings</a>.</p>
```